src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(new Random().nextInt(100),new ColorSupplier().getRandomColor());
```

Comment:
I also recommend making ColorSupplier and Random class fields. And 100 - its a magic namber. make it a constant